From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 11 Jul 2019 20:10:04 +0200
Subject: Revert "meson: Bump meson requirement to 0.50.0"

This reverts commit ccab0f470dcc556073754c8adf9413819d22cc14.

There's not strict requirement on meson 0.50.0, while not all
distros using 3.32 have meson 0.50.0
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index a770b4c..7ed1fb7 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('mutter', 'c',
   version: '3.32.2',
-  meson_version: '>= 0.50.0',
+  meson_version: '>= 0.48.0',
   license: 'GPLv2+'
 )
 
